CAInstructions.markdown

```diff
@@ -106,7 +106,7 @@ Create the insecure key, the one without a passphrase, and interchange the key n
 Using the CSR, generate a certificate signed by the CA:
 
 ```
-    openssl ca -in <usr_common_name>.csr -config openssl.cnf
+    openssl ca -in <usr_common_name>.csr -config <openssl_configuration_file>
```

Comment:
May be use PSC/openssl.cnf ?

As we have used openssl.cnf in Configuration file section:

'''
Create a CA configuration file (refer PSC/openssl.cnf). This is optional, but is very convenient while issuing multiple certificates.

Edit openssl.cnf, and change the following fields: (Here, directory <usr> must be directory TS, CP, or DP depending on the keys you want to generate.)
'''